src/components/PhotoAlbum.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Photos from './Photos';
+import PropTypes from 'prop-types';
+import styles from './PhotoAlbum.css';
+
+
+function PhotoAlbum({ urls }) {
+
+  return (
+    <div className={styles.div}>
```

Comment:
```suggestion
    <div className={styles.album}>
```
I found that naming the style class something closer to the filename kept things a bit more consistent across my files. In this case, I would have maybe named the class 'album' or 'photoablum'.